app/src/main/java/net/gsantner/markor/format/FormatRegistry.java

```diff
@@ -74,6 +76,32 @@ public class FormatRegistry {
     public final static OrgmodeTextConverter CONVERTER_ORGMODE = new OrgmodeTextConverter();
 
 
+
+    public static class Format {
+        public final @StringRes int format, name;
+        public final String ext;
+        public final TextConverterBase converter;
+
+        Format(int format, int name, String ext, final TextConverterBase converter) {
+            this.format = format;
+            this.name = name;
+            this.ext = ext;
+            this.converter = converter;
+        }
+    }
+
+    public static final Format[] FORMATS = new Format[] {
```

Comment:
Name, format, extension and converter are explicitly associated now in this array